stable/aws-vpc-cni/values.yaml

```diff
@@ -8,7 +8,7 @@ nameOverride: aws-node
 
 image:
   region: us-west-2
-  tag: v1.6.1
+  tag: v1.6.2
```

Comment:
You're not just adding `cni-metrics-helper` but you're also bumping the version. Are you sure there's no other required changes for `v1.6.2`? Sometimes other parts of the chart need to change with new CNI releases, e.g. https://github.com/aws/eks-charts/pull/74

Reply:
Never mind, I checked, there's no other changes required: https://github.com/aws/amazon-vpc-cni-k8s/commits/master/config/v1.6/aws-k8s-cni.yaml